lib/Workflow/Persister.pm

```diff
@@ -67,10 +70,13 @@ sub init_random_generators {
 sub init_uuid_generators {
     my ( $self, $params ) = @_;
 
-    eval { require Workflow::Persister::UUID };
-    if (my $msg = $EVAL_ERROR) {
+    try {
+        require Workflow::Persister::UUID
```

Comment:
Should we use Module::Runtime::require_module ?

REF: https://metacpan.org/pod/Module::Runtime#require_module(NAME)

Reply:
See above.